api/src/main/java/com/github/retrooper/packetevents/util/VectorInterface3f.java

```diff
@@ -0,0 +1,7 @@
+package com.github.retrooper.packetevents.util;
+
+public interface VectorInterface3f extends Cloneable {
```

Comment:
This interface should have more methods which are present in both implementstions